README.md

```diff
@@ -53,16 +53,40 @@ POSTing of entries requires sending the following payload object:
 
 ```
 {
-  csrfmiddlewaretoken: '<csrf token>'
-  nonce: '<nonce value obtained from [GET /nonce]>',
-  data: {
-    title: ...
-    ...
+  csrfmiddlewaretoken: required csrf token string obtained from [GET /nonce]
+  nonce: required nonce string obtained from [GET /nonce]
+
+  title: required string
```

Comment:
oops sorry. Just caught this - we should mention that `title`'s max length is 140 characters